cowprotocol/accounting/fees/classified_fees_4058574.sql

```diff
@@ -0,0 +1,60 @@
+-- This query computes fees
+--
+-- It currently uses the raw_order_rewards table which is only available on ethereum. On other chains it returns an empty table.
+--
+-- Parameters:
+--  {{start_time}} - the timestamp for which the analysis should start (inclusively)
+--  {{end_time}} - the timestamp for which the analysis should end (exclusively)
+--  {{blockchain}} - network to run the analysis on
+--
+-- The columns of the result are
+-- - block_time: time of settlement transaction
+-- - tx_hash: settlement transaction hash
+-- - token_address: address of token with a balance change. contract address for erc20 tokens,
+--   0xeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeee for native token
+-- - amount: value of fee in atoms of the token
+-- - transfer_type: 'protocol_fee' for the total protocol fee (including partner fee), 'network_fee' for network fees
```

Comment:
it does look a bit strange that this column is labeled `transfer_type` although the query is meant to just compute fees. Should we maybe name it just `type` and then rename it in the context of slippage accounting?